server/models/Availability.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require("mongoose");
+
+const dateOptions = {
+    startTime: {
+      type: Date, 
```

Comment:
If we're using `Date` let's format it to only store the time instead of the entire date

Reply:
I think im going to change Date to number so that I can easily compare without formatting